rust/datafusion/Cargo.toml

```diff
@@ -40,9 +40,10 @@ name = "datafusion-cli"
 path = "src/bin/main.rs"
 
 [features]
-default = ["cli"]
+default = ["cli", "crypto_expressions"]
```

Comment:
By having them on by default, how do we reduce compile times?

Reply:
> By having them on by default, how do we reduce compile times?

You can locally run the tests with `cargo test --features cli` or something, right?